build.gradle

```diff
@@ -56,6 +56,12 @@ allprojects {
           url "${rootDir}/jcenter-local-m2"
         }
     }
+
+    tasks.withType(org.jetbrains.kotlin.gradle.tasks.KotlinCompile).all {
```

Comment:
Is this just to make the default interface methods work from Java? They work as is in Kotlin right?

Reply:
Yes it's for Java only.